### ISSUE TITLE
Automatically reading related files

### ISSUE THREAD

**Issue by owenzhao** [DanielEdrisian/AlexSideBar-Public#52]
**Describe the feature**
The current approach involves either reading a single file or the entire codebase. However, this can sometimes be inconvenient. For instance, when reading a single file, it's often difficult to understand the data related to a referenced Model. Additionally, one may not be able to read Views in SwiftUI from another file. On the other hand, if one were to read the entire codebase, there would be less necessity for doing so. Moreover, sometimes what is needed might not be a reading of the codebase but rather accessing properties or functionalities provided by a third-party framework or system-integrated frameworks. Otherwise, this could lead to issues with accessing non-existent properties.

**What version are you on currently?**
 - Version 1.3.10 (91)

**Comment by owenzhao** [comment]
Currently, multiple related files can be added to alleviate the issue. However, the process of adding them involves multiple steps, and once they are added, there is no locking functionality. This often leads to incomplete files when switching tabs in Xcode. Additionally, after the current conversation ends, the mode automatically reverts back to a single-file mode in the next round.

**Comment by DanielEdrisian** [comment]
Yeah I love this idea. I have some ideas on how to do this, will release something in the coming weeks
